rotkehlchen/tests/db/test_xpubs.py

```diff
@@ -99,7 +99,7 @@ def fixture_setup_db_for_xpub_tests(data_dir, username):
         xpub=HDKey.from_xpub(xpub=xpub, path='m'),
         derivation_path=derivation_path,
     )
-    data.db.add_bitcoin_xpub(xpub_data3)
+    data.db.add_bitcoin_xpub(xpub_data3, SupportedBlockchain.BITCOIN)
```

Comment:
Can you add a test case for bitcoin cash. I only see the bitcoin param being used